CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
 
+## v1.0.0-rc9
```

Comment:
unfortunatly there are many phase3 changes there so you need to:
- pickup changes from rc.9 and delete them from `Unreleased`
- move them to `v1.0.0-rc9` seciotn

Reply:
good catch

Reply:
fixed